python/projects_and_cores.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""CSBC/PS-ON Project Descriptions
+
+This will scrap for the project (and core) descriptions from each
+CSBC/PS-ON participating institution on Synapse.
+
+author: verena.chung
+"""
+
+import re
+
+import synapseclient
+
+
+def parse_u01_project(page):
+    """Parse U01 project description."""
+
+    page = page.replace("**", "")
+    try:
+        description = re.search(
+            r"Project Description.*?\s(.*?)->", page, re.S).group(1)
+    except AttributeError:
+        description = ""
+
+    return description.rstrip("\n\n&nbsp;").strip()
+
+
+def parse_u54_center(center, page):
+    """Parse project and core descriptions from U54 center."""
+
+    # Center has only project description. Add '#CORE' to the end so that
+    # all project descriptions can be captured.
+    if not re.search(r"P(ROJECT|roject).*#CORE", page, re.S):
+        page += "\n###COR"
+
+    project_subpage = re.search(r"(Project.*)###COR", page, re.S).group(1)
+    projects = re.split(r"\n####\*", project_subpage)
+    project_info = parse_information(center, projects)
```

Comment:
Minor thing, but probably makes more sense to move the function definition for `parse_information` above `parse_u54_center`, as the former is called by the latter.